go.mod

```diff
@@ -153,6 +183,12 @@ require (
 	modernc.org/token v1.0.0 // indirect
 )
 
+replace github.com/vedhavyas/go-subkey => github.com/strangelove-ventures/go-subkey v1.0.7
+
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 replace github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
+
+replace github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
+
+replace github.com/ChainSafe/go-schnorrkel/1 => github.com/ChainSafe/go-schnorrkel v1.0.0
```

Comment:
The replace directives might be okay for now, but replace directives in general do not carry over through dependencies. That is, if `xchain` imports `ibctest`, and if they use ibctest's `chain/polkadot` package, they will fail to import `.../go-schnorrkel/1` unless they copy the replace lines here.

Forcing the importer to copy the replace directives is a clunky solution, but it works.

One alternative would be to fork go-schnorrkel and have both revisions under some `strangelove/schnorrkel` repo. We can do that in a way that retains the git history, so that we can show that the source isn't modified. OTOH this repo is just for test, so the git history of a key encoding library is less important than a production chain.

Reply:
I'm fine forking them to strangelove, but how do we keep them up to date?

Reply:
I doubt the repos are very active, so we would have to set up some manual notification of new commits and write a script to merge in the upstream changes. It would be mildly tedious, but infrequent, so I think the "once in a while, run a script by hand" approach is a better use of time than investing into fully automating it.